src/DropdownMenu/examples/userMenu.scss

```diff
@@ -1,36 +1,36 @@
 @import '../../utils/theme/variables.scss';
 
 .user-menu {
+  padding: 6px 10px;
   height: 50px;
   min-width: 210px;
-  padding: 6px 10px;
 
   .icon {
     font-size: 7px;
   }
 
   .user-name {
+    color: $brc-darkGrey;
     font-size: 16px;
     font-weight: $brc-semibold;
-    color: $brc-darkGrey;
   }
 
   .user-role {
-    font-size: 14px;
     color: $brc-coolGrey;
+    font-size: 14px;
     font-weight: $brc-semibold;
   }
 
   .user-avatar {
     margin-right: 10px;
   }
 
-  & ~ div .menu {
+  ~ div .menu {
```

Comment:
was the `&` causing this css block to be skipped? we should test the final result of this CSS and see if it changes or not